integration/data/test_restore_inc.py

```diff
@@ -132,6 +132,7 @@ def restore_inc_test(grpc_engine_manager,  # NOQA
         for i in range(RETRY_COUNTS):
             rs = cmd.restore_status(dr_address)
             for status in rs.values():
+                assert status['backupURL'] != ""
```

Comment:
Should be?
```
if status['backupURL'] != backup1:
    break
```
So it can sleep for 1 second then retry in case the restored status wasn't updated yet?

Same for the others.